plugins/source/aws/resources/services/quicksight/folders_fetch.go

```diff
@@ -0,0 +1,57 @@
+package quicksight
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/quicksight"
+	"github.com/aws/aws-sdk-go-v2/service/quicksight/types"
+	"github.com/aws/smithy-go"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchQuicksightFolders(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	cl := meta.(*client.Client)
+	svc := cl.Services().Quicksight
+	input := quicksight.ListFoldersInput{
+		AwsAccountId: aws.String(cl.AccountID),
+	}
+	var ae smithy.APIError
+
+	for {
+		out, err := svc.ListFolders(ctx, &input)
+		if err != nil {
+			if errors.As(err, &ae) && ae.ErrorCode() == "UnsupportedUserEditionException" {
```

Comment:
We still get errors like:
```
operation error QuickSight: ListFolders, https response error StatusCode: 403, RequestID: XXXX, AccessDeniedException: User: arn:aws:iam::XXXX:user/XXXX is not authorized to perform: quicksight:ListFolders on resource: arn:aws:quicksight:ap-northeast-2:XXXX:folder/* because no identity-based policy allows the quicksight:ListFolders action"
```